platform/working-with-vsp/orientation/request-access-to-tools.md

```diff
@@ -17,7 +17,7 @@ To work on the Veteran-facing Services Platform, each team member needs to use t
 
 ## GitHub
 
-1. If you haven't already, check your email for an invitation to the Department of Veterans Affairs GitHub Organization, and join! If you have not received an invitation please follow the steps in [Github User Requests](https://github.com/department-of-veterans-affairs/github-user-requests).
+1. If you haven't already, check your email for an invitation to the Department of Veterans Affairs GitHub Organization, and join! If you have not received an invitation (and do not already have a va.gov email), please reach out to your PM and/or team lead and ask that they submit a request for you to be added to the GitHub organization. If you already possess a va.gov email and have not received an invitation, please visit [this site](https://vaww.oit.va.gov/services/github/) (will only work on VA network, CAG, or GFE/VPN) and fill out a request to be added.
```

Comment:
sorry to tweak again, but consider this:
please reach out to your PM and/or team lead and ask that they submit a request for you to be added to the GitHub organization using this link: [Github User Requests](https://github.com/department-of-veterans-affairs/github-user-requests)

As their PM *can* see that link.

Reply:
added in https://github.com/department-of-veterans-affairs/va.gov-team/pull/11950/commits/1796319731cf8cc5c31f13f5ece4e20798d354a1

Reply:
oops, copy pasta
will fix

Reply:
fixed